scripts-dev/lint.sh

```diff
@@ -2,10 +2,8 @@
 #
 # Runs linting scripts over the local Synapse checkout
 # isort - sorts import statements
-# flake8 - lints and finds mistakes
 # black - opinionated code formatter
-
-set -e
```

Comment:
Won't this mean that this script will always return 0?

Reply:
Mmm yes, though I see that it's not actually used in CI.

I believe `flake8` was the only one that would return a non-zero exit code, so as it's at the end now it won't block other programs from executing anymore, and we can keep the `-e`.